src/ray/test/run_gcs_tests.sh

```diff
@@ -16,9 +17,10 @@ bazel build "//:redis_gcs_client_test" "//:redis_actor_info_accessor_test"  "//:
 sleep 1s
 
 ./bazel-bin/redis_gcs_client_test
-./bazel-bin/redis_actor_info_accessor_test
-./bazel-bin/subscription_executor_test
 ./bazel-bin/asio_test
+./bazel-bin/subscription_executor_test
+./bazel-bin/redis_actor_info_accessor_test
+./bazel-bin/object_state_accessor_test
```

Comment:
We shouldn't be adding more tests in this way. Can you make it so that `bazel test` works properly here?

Reply:
+1 for changing this ASAP. And actually we can simplify `BUILD.bazel` by moving everything to one single `redis_gcs_client_test` `cc_test`.

Reply:
I am working on it. But I only found `sh_test` will work now. I am still trying how to run a `setup` `script`  in `cc_test`, maybe use `toolchains `. 
I searched some project that use `bazel`, but not found a clue about use `cc_test` to run a `setup` `script`.

Reply:
@raulchen @micafan and I talked about this today, so far we haven't found a good way to do this yet after doing some research on bazel in the last few days. And actually all similar tests in ray currently use a shell script as well (e.g. node manager, object manager, core worker .etc), so as long as we can keep all the gcs client tests in a single script it doesn't make things worse:)  How about we proceed the GS client PRs and investigate on using `bazel test` separately at the same time? Right now we already have a couple of related PRs pending for a while, I think it would be easier to push forward these PRs to unblock other things. And when we have a good way to use `bazel test,` we can migrate all existing tests to it as well together.  How does this sound?